editor/docks/filesystem_dock.cpp

```diff
@@ -2556,6 +2557,24 @@ void FileSystemDock::_file_option(int p_option, const Vector<String> &p_selected
 			ImportDock::get_singleton()->reimport_resources(p_selected);
 		} break;
 
+		case FILE_MENU_DOCUMENTATION: {
+			for (const String &selected : p_selected) {
+				if (selected.ends_with("/")) {
+					continue;
+				}
+				const StringName file_type = EditorFileSystem::get_singleton()->get_file_type(selected);
+				if (ClassDB::class_exists(file_type)) {
+					if (file_type == "GDScript") {
+						Ref<Script> script_base = ResourceLoader::load(selected);
```

Comment:
Is this guaranteed to return a valid resource? Is it guaranteed to not fail on parse errors etc.?